### DISCUSSION TITLE
Creating an event for external signal

### DISCUSSION THREAD

**Discussion by GyimPet** [blakeblackshear/frigate#9443] [Ideas]
I installed a motion sensor next to the cameras (the reason for this is that there is a lot of light pollution, which is why the image analysis programs often fail)
Is it possible to create an event in Frigate NVR for a motion sensor signal (eg MQTT)?

**Comment by NickM-27** [comment]
This is possible in 0.13

**Comment by GyimPet** [comment]
Ohh ! Thanks for the information ! When is version 0.13 expected to be released ? Where can I read about the planned features ? 

Sorry, if I sound ignorant, but I've only just started to get involved with Frigate.

**Reply by NickM-27** [reply]
there have been many betas and then an RC that was released a couple weeks ago: https://github.com/blakeblackshear/frigate/releases/tag/v0.13.0-rc1

full release is likely coming soon
